stackslib/src/chainstate/nakamoto/tests/mod.rs

```diff
@@ -1636,3 +1643,272 @@ pub fn test_get_highest_nakamoto_tenure() {
+            parent_vtxindex: 1,
+            // miners take turns winning
+            key_block_ptr: miners[i - 3].block_height as u32,
+            key_vtxindex: miners[i - 3].vtxindex as u16,
+            memo: vec![STACKS_EPOCH_3_0_MARKER],
+            commit_outs: vec![],
+
+            burn_fee: 12345,
+            input: (Txid([0; 32]), 0),
+            apparent_sender: BurnchainSigner::mock_parts(
+                AddressHashMode::SerializeP2PKH,
+                1,
+                vec![StacksPublicKey::from_hex(
+                    "02d8015134d9db8178ac93acbc43170a2f20febba5087a5b0437058765ad5133d0",
+                )
+                .unwrap()],
+            ),
+
+            txid: winning_txid.clone(),
+            vtxindex: 1,
+            block_height: snapshot.block_height,
+            burn_parent_modulus: ((snapshot.block_height - 1) % BURN_BLOCK_MINED_AT_MODULUS) as u8,
+            burn_header_hash: snapshot.burn_header_hash.clone(),
+        };
+
+        let winning_ops = if i == 3 {
+            // first snapshot includes leader keys
+            miners
+                .clone()
+                .into_iter()
+                .map(|miner| BlockstackOperationType::LeaderKeyRegister(miner))
+                .collect()
+        } else {
+            // subsequent ones include block-commits
+            if sortition {
+                vec![BlockstackOperationType::LeaderBlockCommit(
+                    winning_block_commit,
+                )]
+            } else {
+                vec![]
+            }
+        };
+
+        {
+            let mut tx = SortitionHandleTx::begin(sort_db, &last_snapshot.sortition_id).unwrap();
+            let _index_root = tx
+                .append_chain_tip_snapshot(
+                    &last_snapshot,
+                    &snapshot,
+                    &winning_ops,
+                    &vec![],
+                    None,
+                    None,
+                    None,
+                )
+                .unwrap();
+            tx.test_update_canonical_stacks_tip(
+                &snapshot.sortition_id,
+                &snapshot.canonical_stacks_tip_consensus_hash,
+                &snapshot.canonical_stacks_tip_hash,
+                snapshot.canonical_stacks_tip_height,
+            )
+            .unwrap();
+            tx.commit().unwrap();
+        }
+
+        last_snapshot = SortitionDB::get_block_snapshot(sort_db.conn(), &snapshot.sortition_id)
+            .unwrap()
+            .unwrap();
+
+        if i > 3 {
+            // have block-commit
+            // check the stackerdb config as of this chain tip
+            let stackerdb_config =
+                NakamotoChainState::make_miners_stackerdb_config(sort_db).unwrap();
+            eprintln!(
+                "stackerdb_config at i = {} (sorition? {}): {:?}",
+                &i, sortition, &stackerdb_config
+            );
+
+            stackerdb_configs.push(stackerdb_config);
```

Comment:
Should these configs be checked at the end of this test?

Reply:
Yes

Reply:
Will do!